Helper/Search/Filter.php

```diff
@@ -167,6 +171,9 @@ public function handleAggregation(array $aggregation)
             if (isset($this->choices[$bucket['key']])) {
                 $this->choices[$bucket['key']]['filter'] = $bucket['doc_count'];
             }
+            if (isset($bucket['reversed_nested'])) {
+                $this->choices[$bucket['key']]['reversed_nested'] = $bucket['reversed_nested']['doc_count'];
```

Comment:
This line is not sure if 'key' is set in $bucket!
Begin the foreach loop maybe with 
if (!isset($this->choices[$bucket['key']])) {
continue;
}
and adapt the other if check.